mne/io/eeglab/eeglab.py

```diff
@@ -1,12 +1,15 @@
-# Author: Mainak Jas <mainak.jas@telecom-paristech.fr>
+# Authors: Mainak Jas <mainak.jas@telecom-paristech.fr>
+#          Jona Sassenhagen <jona.sassenhagen@gmail.com>
 #
 # License: BSD (3-clause)
 
+import warnings
```

Comment:
don't use `warnings.warn()` use `from ...utils import warn` and just do `warn(...)`